packages/design/sass/styles.scss

```diff
@@ -166,3 +166,60 @@ iframe:focus {
   padding-bottom: 62.5%;
   height: 0;
 }
+
+.usa-fieldset {
+  width: 100%;
+
+  .usa-legend,
+  .usa-alert__text {
+    font-size: 1rem;
+  }
+
+  .usa-legend {    
+    font-weight: normal;
+    line-height: 1.5rem;
+    text-transform: uppercase;
+  }
+
+  h3.section-heading-h3 {
+    font-size: 1.375rem;
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
```

Comment:
On all the numeric values, can we use the USWDS token-derived utility classes, or if not possible, the SCSS mixins?

Ideally, there wouldn't be any magic numbers in the styles, and we inherit everything from the USWDS theme. To clarify, I'm referring to these: [Text utilities](https://designsystem.digital.gov/utilities/font-size-and-family/#utility-mixins)

Reply:
I wanted to do that but for some reason the token for numeric value and the ones for colors doesn't work for me.